pagopa/io-app
# [IOCOM-1884] FIMS History autogenerated specs

⚠️ This PR dependes on #6330 ⚠️

## Short description
This PR adds autogenerated specifications for FIMS History.

## List of changes proposed in this pull request
- FIMS History specs linked in package.json with proper command for autogeneration
- Proper types used throughout the FIMS History flow

## How to test
Apart from succeeding static checks, make sure that the FIMS history section retrieves and displays the history accesses and the export work as intended (using the io-dev-api-server)
